CHANGELOG.md

```diff
@@ -1,5 +1,81 @@
 # Changelog
 
+## 0.13 (2020-12-1?)
+
+#### :building_construction: Inline environments
+
+One of the most debated features of the past months has landed. It is now
+possible to define Tanka Environments inline. It is now possible to leverage all
+powerful Jsonnet concepts to modify your Tanka Environments. See
+https://tanka.dev/inline-environments for more details.
+
+#### :wheel_of_dharma: Kustomize support
+
+In 0.12 we brought in [Helm support](#wheel_of_dharma-helm-support), similarly
+Tanka now also comes with Kustomize support.
+
+* We have refactored `helm-util` into
+  [`tanka-util`](https://github.com/grafana/jsonnet-libs/blob/master/tanka-util)
+  to support both `helm.template()` and `kustomize.build()` use cases from a
+  common base.
+* Jsonnet-native overwriting of Kustomizations
+
+Have a look at https://tanka.dev/kustomize on how to use all this goodness.
+Also https://tanka.dev/helm has been updated to match the library changes.
+
+> This feature is currently experimental. We believe it is feature complete, but
+> further usage in the field may lead to adjustments
+
+#### :sparkles: Export `JSONNET_PATH`
+
+Tanka :heart: Jsonnet, and so do you. With this release, you can now access the
+`JSONNET_PATH` that Tanka uses to find all libraries. Try something this in your
+environment:
+
+```console
+$ JSONNET_PATH=$(tk tool jpath environments/prometheus) jsonnet-lint environments/prometheus/main.jsonnet
+```
+
+### Features
```

Comment:
Several PRs below are from non-Grafana contributors. We gave them a shout-out by putting their name behind their PR in other Changelogs, let's do that here as well